### PR TITLE
Add IPv4 DHCP option matcher, minor fixes

### PR DESCRIPTION
**Pull request by hansmi**
##### SUMMARY

RouterOS 7.4 and newer implement matchers for the Vendor ID provided by DHCP clients under `ip dhcp-server matcher`. The default value for `force` on options is also set.

##### ISSUE TYPE
- Feature Pull Request

### CODE REVIEW

File: changelogs/fragments/300-set-ip-dhcp-option-force-default.yml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+minor_changes:
+  - api_info, api_modify - set default for ``force`` in ``ip dhcp-server option`` to an explicit false (https://github.com/ansible-collections/community.routeros/issues/300).
```
Comment (felixfontein) [inline changelogs/fragments/300-set-ip-dhcp-option-force-default.yml:2]: BTW, you can also put both fragments into the same YML file, as two list entries in `minor_changes`.

```suggestion
  - api_info, api_modify - set default for ``force`` in ``ip dhcp-server option`` to an explicit ``false`` (https://github.com/ansible-collections/community.routeros/issues/300).
```

**Review by felixfontein — COMMENTED** [review]
Thanks for your contribution!

File: changelogs/fragments/300-set-ip-dhcp-option-force-default.yml (hunk 1 above)
Comment (hansmi) [reply changelogs/fragments/300-set-ip-dhcp-option-force-default.yml]: Done. I deliberately used separate files because they're added by separate commits.

Comment (felixfontein) [reply changelogs/fragments/300-set-ip-dhcp-option-force-default.yml]: Fine for me. But please note that the commits will get squashed on merge anyway :)

**Review by felixfontein — APPROVED** [review]

**Comment by felixfontein** [comment]
@hansmi thanks!

**Pull request merged**